VoiceOverlay/Controllers/RecordingViewController.swift

```diff
@@ -28,6 +28,7 @@ class InputViewController: UIViewController {
   var autoStopTimer: Timer = Timer()
   
   var speechText: String?
+  var customData: Any?
```

Comment:
Hmm I'm thinking of how we can avoid `Any` here, which might require adding a class or using templating, which might be overkill. So for now, let's keep I guess